packages/oas-form/src/components/fields/MultiSchemaField.js

```diff
@@ -1,57 +1,17 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import * as types from '../../types';
-import {
-  getUiOptions,
-  getWidget,
-  guessType,
-  retrieveSchema,
-  getDefaultFormState,
-  getMatchingOption,
-  // deepEquals,
-} from '../../utils';
+import { getUiOptions, getWidget, guessType, retrieveSchema, getDefaultFormState } from '../../utils';
 
 class AnyOfField extends Component {
   constructor(props) {
     super(props);
 
-    const { formData, options } = this.props;
-
     this.state = {
-      selectedOption: this.getMatchingOption(formData, options),
```

Comment:
Do we ever provide form data to an initial page load? If so, losing this will stop the polymorphism dropdown from having an initially selected value

Reply:
Thankfully we don't. Only data we provide is through JSON Schema `default` props, and those get automatically compiled into the initial page load through `computeDefaults`.